apps/functions-subscription/src/adapters/azure/cosmosdb/subscription.ts

```diff
@@ -0,0 +1,33 @@
+import { pipe } from 'fp-ts/lib/function';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as E from 'fp-ts/lib/Either';
+import { Database } from '@azure/cosmos';
+import {
+  SubscriptionCodec,
+  SubscriptionReader,
+  SubscriptionWriter,
+} from '../../../domain/subscription';
+import { decodeFromItem } from './decode';
+import { cosmosErrorToDomainError } from './errors';
+
+export const makeSubscriptionCosmosContainer = (
```

Comment:
**note**: I used plain cosmos for simplicity, but If it makes sense we can use [CosmosdbModel](https://github.com/pagopa/io-functions-commons/blob/master/src/utils/cosmosdb_model.ts#L178). If so we can do it in a separate task.